source/_posts/2023-01-30-php-core-roundup-9.md

```diff
@@ -0,0 +1,428 @@
+
+	RFC by Nicolas Grekas and Ilija Tovilo, proposing to pass the calling scope to magic accessors to make it trivial to get it. While it is currently possible to retrieve the calling scope by retrieving the backtrace, the RFC proposes to pass the calling scope as the last argument to the magic methods, making it easier compared to fiddling with the debug backtrace.
+
+*	**RFC Under Discussion: [Saner array_(sum|product)()](https://wiki.php.net/rfc/saner-array-sum-product) 💜**
+
+	RFC by George Peter Banyard suggests improvements to the existing array_sum and array_product functions by emitting a warning when the arrays being summed/multiplied contain invalid types. This can introduce backwards-incompatible effects (apart from the warning) on code that relies on the current behavior of the functions.
+
+*	**RFC Under Discussion: [Add file_descriptor() function](https://wiki.php.net/rfc/file-descriptor-function) 💜**
+
+	RFC by George Peter Banyard proposes a new `file_descriptor()` function that returns the integer file descriptor of a given stream, if the underlying file system supports it.
+
+*	**RFC Declined: [Asymmetric Visibility](https://wiki.php.net/rfc/asymmetric-visibility) 💜**
+
+	PHP’s property visibility modifiers (`public`, `protected`, and `private`) apply to both get and set operations. This RFC by Ilija Tovilo and Larry Garfield proposed a new syntax inspired by Swift to asymmetrically declare the visibility modifiers individually for get and set operations:
+
+	```
+	class Foo
+    {
+        public private(set) string $bar;
+    }
+    ```
+	<br>
+	In the snippet above, `public private(set) `indicates that get operations are `public`, while set operations are `private`.
+
+    However, the vote held for this RFC did not pass. A lengthy mailing list discussion on this RFC is available [here](https://externals.io/message/118994).
+ 
+### Notable Mailing List Discussions
+
+* [base64url format](https://externals.io/message/119243)
+* [PHP build for the wasm32-wasi target](https://externals.io/message/119198)
+* [A set of 18 functions/changes to improve PHP core](https://externals.io/message/119238)
+* [PHP support for matrix operations - BLAS, LAPACK](https://externals.io/message/119086)
+* [Unicode Text Processing](https://externals.io/message/119149)
+* [Deprecate ldap_connect with host and port as separate arguments](https://externals.io/message/119425)
+* [Introduce the ability to use the first-call-callable syntax on non-static methods, statically](https://externals.io/message/119392)
+* [Loading SQLite extensions on PDO](https://externals.io/message/119374)
+* [rules for #include directives](https://externals.io/message/119272)
+
+
+## Merged PRs and Commits
+
+Following are some changes that did not go through an RFC process because they are either planned, bug fixes, or progressive enhancements. There are automated unit and integration tests for each of these changes,  the PHP core developers review all pull requests.
+
```

Comment:
It may make sense to highlight outside of the commit list the improvement in performance of MBString by Alex Dowad.

Reply:
You are right, they are noteworthy indeed. I mentioned them above the rest of the commit log. I included Alex's work on mbstring, along with the Random extension and const/include RFC mentions.